### PR TITLE
[FAB-17740] Update configtx package documentation

### PR DESCRIPTION
**Pull request by stephyee**
#### Type of change

- Documentation update

#### Description

Add additional information about the configtx package in the godoc overview 

#### Related issues

[FAB-17740](https://jira.hyperledger.org/browse/FAB-17740)

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -4,7 +4,13 @@ Copyright IBM Corp. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
-// EXPERIMENTAL - Package config allows the creation, retrieval, and modification of channel configtx.
+// EXPERIMENTAL
+//
+// Package config provides utilities to create and modify an channel configuration transaction. Channel transactions contain the configuration data defining members and policies for a system chain (ordering service) or channel.
```
Comment (caod123) [inline pkg/config/config.go]: `modify a`

File: pkg/config/config.go [hunk 2]
```diff
@@ -4,7 +4,13 @@ Copyright IBM Corp. All Rights Reserved.
...
+// A channel transaction can be a transaction that creates a new channel or modifies a existing channel. Both the creation of a new channel or modification of an existing channel outputs an unsigned transaction represented in a protobuf binary format that must be signed by the requisite number of members such that the
```
Comment (caod123) [inline pkg/config/config.go]: `modifies an`

File: pkg/config/config.go [hunk 3]
```diff
@@ -4,7 +4,13 @@ Copyright IBM Corp. All Rights Reserved.
...
+// transaction fulfills the channel's modification policy.
+//
+// See https://hyperledger-fabric.readthedocs.io/en/master/configtx.html#anatomy-of-a-configuration for a in-depth of channel anatomy.
```
Comment (caod123) [inline pkg/config/config.go]: `for an in-depth description of a channel configuration's anatomy`

**Review by caod123 — CHANGES_REQUESTED** [review]

**Comment by wlahti** [comment]
Nit: the lines are really long, which makes it a little annoying to review. :) Any reason to not follow the general standard of <80 characters per line?

**Comment by stephyee** [comment]
@wlahti Not really, I didn't think about it much given its docs and not code. I can shorten them.

File: pkg/config/config.go (hunk 1 above)
Comment (stephyee) [reply pkg/config/config.go]: 🛠️

File: pkg/config/config.go (hunk 3 above)
Comment (stephyee) [reply pkg/config/config.go]: 🛠️

File: pkg/config/config.go (hunk 2 above)
Comment (stephyee) [reply pkg/config/config.go]: 🛠️

File: pkg/configtx/config.go [hunk 4]
```diff
@@ -4,7 +4,16 @@ Copyright IBM Corp. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
-// Package configtx allows the creation, retrieval, and modification of channel configtx.
+// Package config provides utilities to create and modify a channel configuration transaction.
+// Channel transactions contain the configuration data defining members and policies for a
+// system chain (ordering service) or channel. A channel transaction can be a transaction that
```
Comment (caod123) [inline pkg/configtx/config.go]: `Channel transactions contain the configuration data defining members and policies for a system or application channel and can be used to either create or modify existing channels.`

**Review by caod123 — CHANGES_REQUESTED** [review]
Maybe also mention something specific about system channel genesis block? Not sure if it's super necessary though.

**Review by caod123 — APPROVED** [review]

**Pull request merged**